client-data/js/board.js

```diff
@@ -26,6 +26,15 @@
 
 var Tools = {};
 
+Tools.i18n = (function i18n() {
+	var translations = JSON.parse(document.getElementById("translations").text);
+	return {
+		"t": function translate(s) {
+			return translations[s] || s;
+		}
+	};
+})();
+
```

Comment:
Moved this up here, because I got a not-defined warning in Firefox for a line where Tools.i18n was referenced